utils/userFunction.js

```diff
@@ -0,0 +1,107 @@
+const currentDate = new Date();
+export function hasStudiedRecently(wordList) {
+  const currentDate = new Date();
+  if (!wordList) {
+    return;
+  }
+
+  if (wordList) {
+    const hasRecentReview = wordList.some((entry) => {
+      const reviewDate = new Date(entry.study.lastReview);
+      const comparisonDate = new Date(
+        reviewDate.setDate(reviewDate.getDate() + 1)
+      );
+
+      return comparisonDate <= currentDate;
+    });
+
+    if (hasRecentReview) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
You're essentially stating
```js
if (true) {
return true;
}
return false;
```

... [when you could just state](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

```suggestion
    return hasRecentReview;
```

Reply:
True 😅